packages/styles/link.css

```diff
@@ -12,11 +12,10 @@
 .Link {
   text-decoration: underline;
   color: var(--link-text-color);
-  font-weight: var(--font-weight-medium);
-  display: inline-block;
   padding: var(--space-quarter);
   background: transparent;
   vertical-align: baseline;
+  line-height: var(--line-height-body);
```

Comment:
What's the line height change here?

Reply:
I took it from this [comment](https://github.com/dequelabs/cauldron/issues/1672#issuecomment-2344593401)

Reply:
See the description:

> Link typography should generally be inheriting styles from css cascading.

If body has the correct line height, it will cascade down to the link. We don't need to specify the property again, Eric was just clarifying what the general expectation was.